src/presentation/pages/Home/Main/Main.tsx

```diff
@@ -0,0 +1,39 @@
+import { Grid } from '@material-ui/core';
+import { MainText } from './MainText';
+import { MainImg } from './MainImg';
+import Athletes from '../../../../app/assets/Athletes.png';
+import Culinary from '../../../../app/assets/Culinary.png';
+import Music from '../../../../app/assets/Music.png';
+import Ball from '../../../../app/assets/Ball.png';
+import styles from './MainContent.module.scss';
+
+export const MainContent = () => {
+  return (
+    <Grid container className={styles.mainContent}>
+      <Grid item xs={6} md={4} lg={2} className={styles.mainContent__item}>
+        <MainText
+          styles={styles}
+          textContent="Lorem ipsum dolor sit amet, consectetur adipiscing elit"
+          textButton="Know How ↓"
+        />
+      </Grid>
+      <Grid item xs={6} md={4} lg={2} className={styles.mainContent__item}></Grid>
+
+      <Grid item xs={6} md={4} lg={2} className={styles.mainContent__item}>
+        <MainImg src={Athletes} alt="Athletes" styles={styles} link="Athletes" />
```

Comment:
If you are going to add something to `alt` attribute you might want to read [this](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#accessibility_concerns). I think that for these casse you can just put an empty string or try to be more specific with the alt (also you'll want to use i18n for this i think)